### PR TITLE
New fields measuresPeriod and measuresPeriodUnit

### PR DESCRIPTION
**Pull request by VictorVicente1**
Introduction of the new fields measuresPeriod and measuresPeriodUnit to display the measures period and its unit.

### CODE REVIEW

File: Parking/OffStreetParking/doc/spec.md [hunk 1]
```diff
@@ -239,7 +239,15 @@ it conveys what is such special location.
 + `provider` : Parking site service provider.
     + Normative references: [https://schema.org/provider](https://schema.org/provider)
     + Optional
+
++ `measuresPeriod` : The measures period.
```
Comment (jmcanterafonseca) [inline Parking/OffStreetParking/doc/spec.md]: can you please be more precise about what period are you referring to?

File: Parking/OffStreetParking/doc/spec.md [hunk 2]
```diff
@@ -239,7 +239,15 @@ it conveys what is such special location.
...
+    + Attribute type: [Number](http://schema.org/Number)
+    + Optional
     
++ `measuresPeriodUnit` : The measures period unit.
```
Comment (jmcanterafonseca) [inline Parking/OffStreetParking/doc/spec.md]: please check other properties which are using units of measurement to check how is this conveyed in the data models

thanks

**Comment by jmcanterafonseca** [comment]
@VictorVicente1 any news about updating this PR?

thanks!

**Comment by VictorVicente1** [comment]
@jmcanterafonseca  It was everything collapsed in #133

**Pull request closed without merging**